hack/release_notes/go.mod

```diff
@@ -0,0 +1,10 @@
+module sigs.k8s.io/krew/hack/release_notes
```

Comment:
Idea: why not make this a sub-module on minikube that we can `go get`?

Reply:
Now that's a good idea! I'll open an issue and see if they are open for this.